haralyzer/assets.py

```diff
@@ -407,7 +407,7 @@ def time_to_first_byte(self):
         initial_entry = self.entries[0]
         ttfb = 0
         for k, v in iteritems(initial_entry['timings']):
-            if k != 'receive':
+            if k != 'receive' and isinstance(v, int)
```

Comment:
BTW, you can check the build status here to make sure that your changes are passing the tests:

https://travis-ci.org/mrname/haralyzer